tests/unit/test_table_service.py

```diff
@@ -19,9 +29,10 @@ def test_create_external_table_happy_path(
     cursor_mock = MagicMock()
     connection.cursor.return_value = cursor_mock
 
-    ts = TableService(mock_table, connection)
-    mock_table.compression = "GZIP"
-    ts.create_external_table(mock_aws_settings)
+    tb = request.getfixturevalue(table)
```

Comment:
Just curious, why do you use `getfixturevalue` instead of the variable value itself? Does it make any difference here?

Reply:
I use parametrize which are strings so when you use let's say table directly, it contains a string mock_table, not a mock itself. So that was the way to get to the mock's value.

@pytest.mark.parametrize(
    "aws_settings, table, s3_url",
    [
        ("mock_aws_settings", "mock_table", "mock_s3_url"),
        ("mock_aws_settings_without_s3_url", "mock_table_with_s3_url", "mock_s3_url_1"),
        ("mock_aws_settings", "mock_table_with_s3_url", "mock_s3_url_1"),
    ],
)